package.json

```diff
@@ -23,18 +23,22 @@
     "url": "https://github.com/omnidan/node-emoji/issues"
   },
   "dependencies": {
-    "@sindresorhus/is": "^5.3.0",
-    "char-regex": "^2.0.1",
+    "@sindresorhus/is": "^3.1.2",
+    "char-regex": "^1.0.2",
     "emojilib": "^2.4.0",
-    "skin-tone": "^3.0.0",
+    "skin-tone": "^2.0.0",
```

Comment:
@omnidan I'd kept these at the lower ranges in #113 and #124 because the larger numbers don't work for CJS environments. `@swc/jest` is acting like that. Was it intentional to bump them up?